src/encoded/static/components/award.js

```diff
@@ -113,7 +141,11 @@ function createDoughnutChart(chartId, values, labels, colors, baseSearchUri, nav
                         if (activePoints[0]) { // if click on wrong area, do nothing
                             const clickedElementIndex = activePoints[0]._index;
                             const term = chart.data.labels[clickedElementIndex];
-                            navigate(`${baseSearchUri}${globals.encodedURIComponent(term)}`);
+                            if (chart.options.onClick.baseSearchUri) {
```

Comment:
I don't understand what this IF statement is doing.  How does chart.options.onClick.baseSearchURi get set, and why would it differ from ${baseSearchUri}???